packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -393,122 +494,160 @@ function DragIntoListExample() {
-          break;
-      }
-    }
-    sourceList.remove(...keys);
-  };
-
-  let {dragHooks} = useDnDHooks({
-    getItems(keys) {
-      return [...keys].map(key => {
-
-        let item = sourceList.getItem(key);
-        // Setup the info for each item that will be propagated upon drag
-        return {
-          [`${dragType}-${item.type}`]: JSON.stringify(item)
-        };
-      });
-    }
-  });
-
-  let {dropHooks} = useDnDHooks({
-    async onDrop(e) {
-      let keysToMove = [];
-      // Only add the dropped item to the list of keys to move
-      // if the dropped item has the expected types
-      for (let item of e.items) {
-        if (item.kind === 'text' && item.types.has(acceptedDragTypes)) {
-          let type = acceptedDragTypes;
-          let {id} = JSON.parse(await item.getText(type));
-          keysToMove.push(id);
-        }
-      }
-      onMove(keysToMove, e.target);
-    },
-    getDropOperation(target, types) {
-      let typesSet = types.types ? types.types : types;
-      let draggedTypes = [...typesSet.values()];
-      // Cancel the drop operations if any of drag items aren't a file or if attempting to drop on a item
-      if (!draggedTypes.every(type => acceptedDragTypes === type) || (target.type === 'item' && target.dropPosition === 'on')) {
-        return 'cancel';
-      }
-
-      return 'move';
-    }
-  });
 
   return (
     <Flex wrap gap="size-300">
-      <ListView
-        aria-label="Draggable ListView in drag into list example"
-        selectionMode="multiple"
-        width="size-3600"
-        height="size-2400"
-        items={sourceList.items}
-        dragHooks={dragHooks}>
-        {item => (
-          <Item textValue={item.name}>
-            <Text>{item.name}</Text>
-          </Item>
-        )}
-      </ListView>
-      <ListView
-        aria-label="Droppable ListView in drag into list example"
-        width="size-3600"
-        height="size-2400"
-        items={targetList.items}
-        dropHooks={dropHooks}>
-        {item => (
-          <Item textValue={item.name}>
-            <Text>{item.name}</Text>
-          </Item>
-        )}
-      </ListView>
+      <DraggableListView list={sourceList} />
+      <DroppableListView list={targetList} />
     </Flex>
   );
 }
 ```
 </details>
 
 ```tsx example
-<DragIntoListExample />
+<DragIntoList />
 ```
 
-The example below replicates the previous example but demonstrates how to handle multiple item types. In the droppable ListView, folders are valid drop targets, but we cancel any
-drop operations if any of the dragged items is a folder.
+### Handling folder drops
+
+The example below replicates the previous example but demonstrates how to handle on item drops.
 
 <details>
   <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
 
 ```tsx example export=true render=false
-function DragIntoListFilesOnly() {
+function DraggableListViewFolder(props) {
```

Comment:
I only highlighted the areas of code that differed heavily from the previous example or were the key dnd option/change for the current example. This means there really isn't too much highlighted, lemme know if we'd rather highlight all the DnD related code in each example

Reply:
Yep I think this is good!